e2e_test/batch/aggregate/jsonb_agg.slt.part

```diff
@@ -0,0 +1,88 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create table t(v1 boolean, v2 int, v3 varchar, v4 jsonb);
+
+statement ok
+insert into t values
+    (null, null, null, null),
+    (false, 1, 'ccc', 'null'),
+    (true, 2, 'bbb', 'false'),
+    (true, 3, 'aaa', '{}');
+
+query T
+select jsonb_agg(v1) from t;
+----
+[null, false, true, true]
+
+query T
+select jsonb_agg(v2) from t;
+----
+[null, 1, 2, 3]
+
+query T
+select jsonb_agg(v2::int2) from t;
+----
+[null, 1, 2, 3]
+
+query T
+select jsonb_agg(v2::int8) from t;
+----
+[null, 1, 2, 3]
+
+query T
+select jsonb_agg(v2::float4) from t;
+----
+[null, 1.0, 2.0, 3.0]
+
+query T
+select jsonb_agg(v2::float8) from t;
+----
+[null, 1.0, 2.0, 3.0]
+
+query T
+select jsonb_agg(v3) from t;
+----
+[null, "ccc", "bbb", "aaa"]
+
+query T
+select jsonb_agg(v4) from t;
+----
+[null, null, false, {}]
+
+# query T
+# select jsonb_agg(distinct v1) from t;
+# ----
+# [true, null, false]
```

Comment:
Why this is commented out?

Reply:
I thought the order would be unstable but actually it's not.
However, I still have no idea why the result is not `[null, false, true]`. 🤔